packages/components/src/components/data-list/data-list.scss

```diff
@@ -93,6 +93,23 @@
     }
 }
 
+.statement__row--detail-text-mobile {
+    width: calc(100vw - 3.2rem);
+    height: 100%;
+    padding: 0.6rem 1.6rem;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+    line-height: 1.43;
```

Comment:
do we need 'line_height' here?

Reply:
It was in the design, but if I leave it out the default is `1.40`. Shouldn't make a difference, right?